src/bin/pg_autoctl/cli_show.c

```diff
@@ -922,7 +921,6 @@ cli_show_uri_getopts(int argc, char **argv)
 
 	static struct option long_options[] = {
 		{ "pgdata", required_argument, NULL, 'D' },
-		{ "monitor", required_argument, NULL, 'm' },
```

Comment:
Do we really want to remove the `--monitor` option from the command? It should be possible still to do the following:

```
$ pg_autoctl show uri --monitor postgres://monitor:5432/pg_auto_failover --formation foo
```

I suppose it's only documentation that should be updated.

Reply:
Alright, --monitor option has been put back.